### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by adrianclay**
Remove to reduce confusion

### CODE REVIEW

**Review by duncanc19 — APPROVED** [review]

File: src/lib/fragments/UkGovDashComponents.react.js [hunk 1]
```diff
@@ -1,41 +0,0 @@
-import React, {Component} from 'react';
```
Comment (tom-clarke-dluhc) [inline src/lib/fragments/UkGovDashComponents.react.js:1]: Why is this file no longer needed? (but the fragment is?)

Comment (Jordan-Hall) [reply src/lib/fragments/UkGovDashComponents.react.js:1]: Because it qas the base file that was generated when you use the dash react generator

Comment (tom-clarke-dluhc) [reply src/lib/fragments/UkGovDashComponents.react.js:1]: Good to know! Thanks Jordan

Comment (adrianclay) [reply src/lib/fragments/UkGovDashComponents.react.js:1]: Doh, I see what you mean.  Yeah I thought there wasn't a component when I checked.  How wrong I was.

**Pull request merged**